Update google_translate to include dialects (#25113)
home-assistant/home-assistant.github.io: source/_integrations/google_translate.markdown

  required: false
  type: string
  default: "`en`"
tld:
  description: "The default Google domain you want to use to choose dialect."
  required: false
  type: string
  default: "`com`"
{% endconfiguration %}

Check the [complete list of supported languages](https://translate.google.com/intl/en_ALL/about/languages/) (languages where "Talk" feature is enabled in Google Translate) for allowed values.

> Use the 2 digit language code which you can find at the end of URL when you click on Language name. 

Reviewer: ```suggestion
Use the 2-digit language code which you can find at the end of the URL when you click on the language name. 
```

> Check the [complete list of supported tld](https://www.google.com/supported_domains) for allowed TLD values. This is used to force the dialect used when multiple fall into the same 2 digit language code(ie, *US, UK, AU*)

Reviewer: ```suggestion
Check the [complete list of supported tld](https://www.google.com/supported_domains) for allowed TLD values. This is used to force the dialect used when multiple fall into the same 2-digit language code(i.e., *US, UK, AU*)
```

> You can also use supported BCP 47 tags like the below or the 2-2 digit format for your supported dialect(`en-gb` or `en-us`). Below is a list of the currently implemented mappings:

Reviewer: ```suggestion
You can also use supported BCP 47 tags like the below or the 2-2 digit format for your supported dialect(`en-gb` or `en-us`). Below is a list of the currently implemented mappings:

```

> You can also use supported BCP 47 tags like the below or the 2-2 digit format for your supported dialect(`en-gb` or `en-us`). Below is a list of the currently implemented mappings:

Reviewer: ```suggestion
You can also use supported BCP 47 tags like the below or the 2-2 digit format for your supported dialect(`en-gb` or `en-us`). Below is a list of the currently implemented mappings:

```

tts:
  - platform: google_translate
    language: "de"
    tld: com
```

## Service say

> The `google_translate_say` service support `language` and also `options` for set, `tld`. The text for speech is set with `message`. Since release 0.92, service name can be defined in configuration `service_name` option.

Reviewer: ```suggestion
The `google_translate_say` service supports `language` and also `options` for setting `tld`. The text for speech is set with `message`. Since release 0.92, the service name can be defined in the configuration `service_name` option.
```

Say to all `media_player` device entities:

```yaml
# Replace google_translate_say with <platform>_say when you use a different platform.
service: tts.google_translate_say

> data:
>   entity_id: all

Reviewer: ```suggestion
target:
  entity_id: all
data:
```
